src/components/Navbar.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useContext } from 'react';
+import {Link} from 'react-router-dom';
+import { ThemeContext } from '../context/themeContext';
+import {useAppSelector} from '../hooks/reduxHooks'
+
+
+
+const Navbar = () => {
+    const {theme, toggleTheme} = useContext(ThemeContext);
+    const count = useAppSelector((state) => state.count); 
+
+    if(theme.background === "#fff")
```

Comment:
Two things here:
1. Check the name of the theme instead of the color of a property contained within the theme.
2. I'm not sure why you are checking the background color here. Your theme should change when you toggle it and all of your colors will change automatically and you wouldn't need to duplicate your code. I bet if you removed this entire if/else condition and just kept one return statement, it would work fine.